site/en/blog/extension-instantnav/index.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "Chrome Extensions: Extending API to support Instant Navigation"
+description: >
+    The Extensions API has been updated to support back/forward cache, preloading navigations.
+authors:
+  - dtapuska
+hero: 'image/zKrSUSkPboWMTTSEkowJbqw5Egi2/r5IdmaTdE5wBS2d6CCQY.png'
+alt: >
+  Instant navigation in a tab.
+date: 2022-10-04
+---
+
+TL;DR: You may need to update your extension if it assumes `frameId == 0`
+is a page’s main frame. See below for details.
```

Comment:
There is a lot more in this article than `frameId`.  The text from the description would be better here.

Reply:
Simeon wrote the TLDR, but changed to description I wrote.